templates/slurm_stats.sh.epp

```diff
@@ -28,10 +28,14 @@ echo "slurm_nodesumdata,partition=all,type=total count=${nodes_total}"
 total_jobs=$(cat $tf | wc -l)
 running_jobs=$(cat $tf | grep RUNNING | wc -l )
 pending_jobs=$(cat $tf | grep PENDING | wc -l )
+max_jobs=$("${slurm_path}"/scontrol show config | grep MaxJobCount | cut -d' ' -f 15)
```

Comment:
We can't really predict how Slurm might change, but it seems like using something like `awk '{print $3}'` might be less likely to break than counting spaces.

Your call on whether to change this or not.

Reply:
I'll leave for now since it matches what we had before; but I've added this to my list of things for the refactoring of this script that I have in my queue of things to do.